src/utils/validators/graphqlQueryValidators.ts

```diff
@@ -84,15 +83,7 @@ export const createDonationQueryValidator = Joi.object({
     .required()
     .valid(...Object.values(NETWORK_IDS)),
   tokenAddress: Joi.string().pattern(ethereumWalletAddressRegex),
-  token: Joi.string()
-    .required()
-    .pattern(tokenSymbolRegex)
```

Comment:
Just remove the pattern matching and please keep the error message

Reply:
Or keep it as commented!

Reply:
Done

Reply:
Sorry, I didn't mean commenting out the error message!